### ISSUE TITLE
Error with Fable 3 and Webpack

### ISSUE THREAD

**Issue by njlr** [Zaid-Ajaj/Fable.SimpleJson#64]
I get the following error using Webpack dev server:

```
Uncaught TypeError: tree is undefined
    Convert_usingFable3 Json.Converter.fs:31
    <anonymous> Json.Converter.fs:33
   ...
```

Perhaps the Fable 3 test is missing a check?

This...

```fsharp
    let usingFable3() =
        let map = JS.JSON.parse(JS.JSON.stringify (Map.ofList [ 1, 1; 2, 2 ]))
        let tree = get "tree" map
        isDefined (get "k" tree) && isDefined (get "v" tree) && isDefined (get "h" tree)
```

... becomes...

```fsharp
    let usingFable3() =
        let map = JS.JSON.parse(JS.JSON.stringify (Map.ofList [ 1, 1; 2, 2 ]))
        let tree = get "tree" map
        isDefined tree && isDefined (get "k" tree) && isDefined (get "v" tree) && isDefined (get "h" tree)
```

Fable `3.1.12`
SimpleJson `3.18.1`

**Comment by njlr** [comment]
Downgrading to Fable `3.1.5` seems to solve the issue

**Comment by Zaid-Ajaj** [comment]
Hi @njlr thanks for reporting the issue, the fix is actually much easier right now because there should be a `FABLE_COMPILER_3` attribute that we can use. I will fix this later tonight

**Comment by Zaid-Ajaj** [comment]
Fixed as of v3.19 it should work in latest Fable as well

**Comment by tforkmann** [comment]
Awesome! Thanks
